### PR TITLE
[SDTEST-171] add Ruby to the list of supported languages of Datadog Jenkins plugin

### PR DESCRIPTION
**Pull request by anmarchenko**
### What does this PR do? What is the motivation?
Adds Ruby to the list of supported languages for Jenkins Plugin. Updates Jenkins Plugin settings screenshot.

### Merge instructions

Merge readiness:
- [x] Ready for merge

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: static/images/ci/ci-jenkins-plugin-tests-config.png
Comment (janine-c) [inline static/images/ci/ci-jenkins-plugin-tests-config.png:1]: Hi @anmarchenko 👋🏻 For image updates, we sometimes run into caching issues when you update images with the same filename. Could you add the new image as `ci-jenkins-plugin-tests-config-2.png`, and update the reference to that image with the new filename? (We have a script that will then delete the old version of the image for you.) Thank you!

**Review by janine-c — CHANGES_REQUESTED** [review]

File: static/images/ci/ci-jenkins-plugin-tests-config.png
Comment (anmarchenko) [reply static/images/ci/ci-jenkins-plugin-tests-config.png:1]: yes, will do that!

Comment (anmarchenko) [reply static/images/ci/ci-jenkins-plugin-tests-config.png:1]: @janine-c done

**Review by romainkomorndatadog — APPROVED** [review]
LGTM

**Review by drodriguezhdez — APPROVED** [review]

**Review by janine-c — APPROVED** [review]

**Pull request merged**